### PR TITLE
Add support for iPhones and iPads of 2021

### PR DESCRIPTION
**Pull request by LisonFan**

### CODE REVIEW

**Comment by Apanlin** [comment]
谢谢，哥们牛批

File: LFGBDeviceInfo.podspec [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
-  s.name                      = 'GBDeviceInfo'
-  s.version                   = '6.6.0'
+  s.name                      = 'LFGBDeviceInfo'
+  s.version                   = '6.6.1'
   s.summary                   = 'Detects the hardware, software and display of the current iOS or Mac OS X device at runtime.'
   s.author                    = 'Luka Mirosevic'      
   s.homepage                  = 'https://github.com/lmirosevic/GBDeviceInfo'
   s.license                   = { :type => 'Apache License, Version 2.0', :file => 'LICENSE' }
-  s.source                    = { :git => 'https://github.com/lmirosevic/GBDeviceInfo.git', :tag => s.version.to_s }
```
Comment (ricsantos) [inline LFGBDeviceInfo.podspec:8]: why this change?

**Comment by ricsantos** [comment]
Nice work with the PR! Apart from the source change to the podspec, looks good!

If you could revert that line, maybe this will get merged.

**Comment by LisonFan** [comment]
> Nice work with the PR! Apart from the source change to the podspec, looks good!
> 
> If you could revert that line, maybe this will get merged.

Very sorry, have reset those two commits

**Comment by lmirosevic** [comment]
Thanks everyone! Merging this.

**Comment by lmirosevic** [comment]
Published the Pod as 6.7.0
